lib/dcu.robot

```diff
@@ -0,0 +1,148 @@
+*** Settings ***
+Documentation       Library for using the Dasharo Configuration
+...                 Utility tool to manage a firmware binary file.
+
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             SSHLibrary
+Resource            ../keywords.robot
+
+
+*** Variables ***
+${DCU_REPO}=    https://github.com/Dasharo/dcu
+
+
+*** Keywords ***
+DCU Smbios Set UUID In File
+    [Documentation]    Use DCU to set the UUID in a firmware file
+    [Arguments]    ${fw_file}    ${uuid}
+    Run    git clone ${DCU_REPO}
```

Comment:
Cloning over and over seems troublesome.

There is parallel effort from @EduKav1813 on replacing this with submodule: https://github.com/Dasharo/open-source-firmware-validation/pull/611/files

Can you both discuss together how you can integrate these changes?

Reply:
It might be enough if you pick the submodule commit to your PR and remove all of the `git clone` calls.